### PR TITLE
Add WithdrawStakeView + WithdrawStake component

### PR DESCRIPTION
**Pull request by veado**
- Refactor folder structure + filenames to be more in common

In preparation for #533

### CODE REVIEW

File: src/renderer/components/pool/PoolDetails.tsx [hunk 1]
```diff
@@ -103,3 +103,5 @@ export const PoolDetails: React.FC<Props> = ({
     </Styled.Container>
   )
 }
+
+export default PoolDetails
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/pool/PoolDetails.tsx:107]: may we rethink using of default exports in the application ? in general people around suppose that this is not a good idea...
https://basarat.gitbook.io/typescript/main-1/defaultisbad

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

https://medium.com/@rajeshnaroth/avoid-es6-default-exports-a24142978a7a

**Review by thatStrangeGuyThorchain — APPROVED** [review]

File: src/renderer/components/pool/PoolDetails.tsx (hunk 1 above)
Comment (veado) [reply src/renderer/components/pool/PoolDetails.tsx:107]: Valid point :+1:

Comment (veado) [reply src/renderer/components/pool/PoolDetails.tsx:107]: _I'm going to revert these kind of changes with next PR..._

Comment (veado) [reply src/renderer/components/pool/PoolDetails.tsx:107]: ^  https://github.com/thorchain/asgardex-electron/pull/536

**Pull request merged**